src/utils/helpers/utilityFunctions.ts

```diff
@@ -29,7 +29,8 @@ export const substituteValues = (
   // Custom replacement function for regex replace
   const getObjectProperty = (_: string, __: string, property: string) => {
     if (typeof data !== 'object') return data
-    return extractObjectProperty(data, property, `Can't find property: ${property}`)
+    const value = extractObjectProperty(data, property, `Can't find property: ${property}`)
+    return value ?? ''
```

Comment:
Unrelated, but small tweak so that `null` values don't return "null" string.

Reply:
Ok - nice!

Another string that should possibly be localised....

Reply:
Probably not -- should only ever be seen in development.

And if it does ever show up in production, then users will be more likely to file a bug report if it's in the wrong language! :)